CONTRIBUTING.md

```diff
@@ -0,0 +1,159 @@
+
+After startup and finishing cloning of projects, your will have
+- che-sidecar-kubernetes-tooling
+- che-theia
+- vscode-kubernetes-tools
+
+
+## Contribute to VS Code Kubernetes tooling extension
+
+The VS Code [Kubernetes tooling extension](https://github.com/Azure/vscode-kubernetes-tools) will be cloned to your workspace in `vscode-kubernetes-tools` directory.
+You can easily create branch, code, commit and push your changes using using Che-Theia editor.
+
+To create branch, commit and push your changes use Source Control view of Che-Theia or command line git by opening a terminal in `theia-ide` or `che-dev` container.
+
+Tu build and run the extension you can use set of predefined commands.
+
+### Install node dependencies
+
+To install node dependencies for `vscode-kubernetes-tools`, click `'1.1 Kubernetes Plugin :: Install dependencies'` command in `MY WORKSPACE` view which on the right. It will run `npm install` command in `/projects/vscode-kubernetes-tools` directory iside `che-dev` container.
+
+[che-dev]
+```
+$ cd /projects/vscode-kubernetes-tools
+$ npm install
+```
+
+### Package the extension
+
+To pack your extension to a `.vsix` file use `'1.2 Kubernetes Plugin :: Package'` command. It will run [vscode etxnension packager](https://github.com/microsoft/vscode-vsce) in `che-dev` container. An apropriate `vscode-kubernetes-tools-{version}.vsix` will appear in the project directory when success.
+
+[che-dev]
+```
+$ cd /projects/vscode-kubernetes-tools
+$ vsce package
+```
+
+### Build Che-Theia remote plugin connector
+
+[Che-Theia](https://github.com/eclipse/che-theia) repository will be cloned to your workspace on startup.
+I contains a special [eclipse-che-theia-plugin-remote](https://github.com/eclipse/che-theia/tree/master/extensions/eclipse-che-theia-plugin-remote) extension, which is used to launch VS Code extensions as a remote plugin by running it in another container, different from `theia-ide`. To build [eclipse-che-theia-plugin-remote](https://github.com/eclipse/che-theia/tree/master/extensions/eclipse-che-theia-plugin-remote) extension use `'1.3 Che-Theia plugin-remote :: Compile'` command. It will run the following inside `che-dev` container.
+
+[che-dev]
+```
+$ cd /projects/che-theia/extensions/eclipse-che-theia-plugin-remote
+$ yarn
+```
+
+### Run Che-Theia + VS Code Kubernetes tooling extension
+
+The `buildah-dev` container is used to run the kubernetes tooling extension. Projects (`/projects`) directory is shared between containers, and the built `eclipse-che-theia-plugin-remote` is accesisble from `buildah-dev`.
+To run the extension use `'2.1 Run :: Remote Kubernetes extension'` command. The command will configure necessary environment variables, download the latest 0.7.2 release of [redhat.vscode-yaml](https://github.com/redhat-developer/vscode-yaml/releases) extension on which the Kubernetes tooling depends, and will run `eclipse-che-theia-plugin-remote`.
```

Comment:
If you reference version 0.7.2 version here it means you'll have to come back and update the doc whenever that version changes. Could you say latest or just leave it with no version spec?

Reply:
latest is better, done.